### PR TITLE
Specs for Swift Log files

### PR DESCRIPTION
**Pull request by akshay196**
Specs for `/var/log/containers/swift/swift.log` and
`/var/log/swift/swift.log`

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -2539,6 +2539,16 @@
             ],
             "symbolic_name": "ssh_foreman_proxy_config"
         },
+        {
+            "file": "/var/log/containers/swift/swift.log",
+            "pattern": [],
+            "symbolic_name": "swift_log"
+        },
+        {
+            "file": "var/log/swift/swift.log",
```
Comment (bfahr) [inline uploader.json]: @akshay196 your missing a leading `/` here.

File: uploader.v2.json [hunk 2]
```diff
@@ -2534,6 +2534,16 @@
             ],
             "symbolic_name": "ssh_foreman_proxy_config"
         },
+        {
+            "file": "/var/log/containers/swift/swift.log",
+            "pattern": [],
+            "symbolic_name": "swift_log"
+        },
+        {
+            "file": "var/log/swift/swift.log",
```
Comment (bfahr) [inline uploader.v2.json]: Same comment as above.

File: uploader.json (hunk 1 above)
Comment (akshay196) [reply uploader.json]: @bfahr Fixed. Thanks for pointing.

**Comment by SteveHNH** [comment]
@akshay196 @bfahr  Can you fix the conflicts in this one? I'm usually good fixing minor stuff, but I'd rather not adjust the actual specs without knowing what should be changed.
Thanks!

**Comment by bfahr** [comment]
@SteveHNH sorry I didn't see your comment until today.  @akshay196 I fixed the conflict, but looks like we'll need to wait until next production release.

**Pull request merged**